pkg/mtbroker/ttl.go

```diff
@@ -19,6 +19,9 @@ package broker
 import (
 	"context"
 
+	cloudeventsv2 "github.com/cloudevents/sdk-go/v2"
+	clientv2 "github.com/cloudevents/sdk-go/v2/client"
+
 	cloudevents "github.com/cloudevents/sdk-go"
```

Comment:
Why you didn't removed the v1?

Reply:
I thought there can be downstream projects that are using them.
Can we get rid of it without deprecating first?